ndcube/utils/wcs.py

```diff
@@ -270,6 +269,17 @@ def _wcs_slicer(wcs, missing_axis, item, numpy_order=True):
     new_wcs = wcs.slice(item_, numpy_order=False)
     return new_wcs, missing_axis, dropped_coords
 
+def _find_keys_in_wcs_ivoa_mapping(missing_axis):
```

Comment:
I think this function shouldn't just return the keys, but return the value associated with a user-supplied key.  See how it's done here given a single key, denoted byt ```ctype[i]```: https://github.com/sunpy/ndcube/blob/master/ndcube/ndcube.py#L235

Reply:
No problem... fixed it.